### PR TITLE
Feature | Ausbau Stufen Gradle Plugin

### PR DESCRIPTION
**Pull request by ExLeonem**
### Description
- update documentation
  - switch from `projectStage` to `status`
  - add migration guide
- update config file to use new status

#### Old plugin tasks still working

1. Try the following tasks with the old plugin version
   - `uploadFormularFiles`
   - `uploadProcessModelFiles`
   - Tasks should still work when using `projectStage`

#### Switch from `projectStage` to `status` working

1. Switch to newest Plugin Version
2. Execute tasks
   - `uploadFormularFiles`
   - `uploadProcessModelFiles`
   - Tasks should work with the new `status` attribute in default.config

### Deployment-Locations

Check other PR's in bitbucket for deployment locations

### CODE REVIEW

File: migration.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+   2. [Mapping Projektstufen auf neue Statuswerte](#mappingalte-stufen-auf-neue-statuswerte)
+
+## Umstellung Projektstufen zu Statuswerten
+
+Mit der aktuellsten version des Gradle Plugins wurden die Projektstufen entfernt und durch
+Statuswerte ersetzt. 
+Die Schnittstelle selbst funktioniert übergangsweise sowohl mit der Stufe, als auch dem neuen Status.
+Es darf jedoch nur einer der beiden Werte an die Schnittstelle übertragen werden.
+
+**Wichtig:** Ab einem einem Zeitpunkt X, wird die Verwendung der Projektstufen vollständig ausgebaut
```
Comment (mattbond) [inline migration.md]: Perhaps use wording like "deprecated"

File: migration.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| CERTIFIED         | FINAL  |
+
+#### Prozesse
+
+| Projektstufe             | Status |
+|--------------------------|--------|
+| FUNCTIONAL_ANALYSIS      | EDIT   |
+| TECHNICAL_IMPLEMENTATION | EDIT   |
+| QUALITY_ASSURANCE        | TEST   |
+| CERTIFIED                | FINAl  |
```
Comment (mattbond) [inline migration.md]: Typo FINAl  -> FINAL

**Review by mattbond — CHANGES_REQUESTED** [review]

**Comment by ExLeonem** [comment]
Erst Mergen wenn die neuste Version des Plugins selbst ausgeliefert wurde.

**Review by mattbond — APPROVED** [review]

**Comment by mattbond** [comment]
The version number of the plug-in in build.gradle needs to be updated one published.

**Pull request closed without merging**